.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -6,8 +6,10 @@ labels: "\U0001F41E bug"
 assignees: ''
 
 ---
-
 ## Steps to reproduce
+[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
+[comment]: # (Every issue should have exactly bug described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
```

Comment:
"have the exact bug and steps to reproduce described in it"

Reply:
ohh, its supposed to be "exactly one bug". ill correct it